### PR TITLE
feat:-add rating component

### PR DESCRIPTION
**Pull request by shubhamkumar648**
rating comonent
-basic component
Screenshot from 2022-02-11 22-30-14
Screenshot from 2022-02-11 23-56-51

### CODE REVIEW

**Review by SimranjitKaurMaan — COMMENTED** [review]
You can use lighter shade to indicate the unchecked rating icon.

File: styles/rating.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.rating_main_container {
```
Comment (vaibhav-mahalle) [inline styles/rating.css:1]: no apparent changes required.

**Review by vaibhav-mahalle — APPROVED** [review]
nice descriptive class names. overall looks good to me.

**Comment by shubhamkumar648** [comment]
> You can use lighter shade to indicate the unchecked rating icon.



> You can use the lighter shade to indicate the unchecked rating icon.

Okay !  i will do it using  on e-commerce webpage

File: component/rating.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+              style="
```
Comment (rahulcs754) [inline component/rating.html:108]: don't use inline css

**Review by rahulcs754 — COMMENTED** [review]
Don't use inline HTML always use class css

File: component/rating.html (hunk 2 above)
Comment (shubhamkumar648) [reply component/rating.html:108]: this is carbon sh , so it crete link auto with inline style, otherwise it make issue on responsive

**Review by rahulcs754 — APPROVED** [review]

**Pull request merged**